README.md

```diff
@@ -1 +1,5 @@
-WIP
\ No newline at end of file
+# iota-sdk
+
+[![Coverage Status](https://coveralls.io/repos/github/Alex6323/iota-sdk/badge.svg?branch=chore/tests/increase-coverage-1)](https://coveralls.io/github/Alex6323/iota-sdk?branch=chore/tests/increase-coverage-1)
```

Comment:
Should use the iotaledger repo and develop branch?

Reply:
yup ... will be changed in the last commit

Reply:
Done